CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
+v2.15.0
+=======
+* added setting default time during conversion from date to time storing formats in `Converter`. Missing time values are filled with zeros (e.g. `LocalDateTime` format specified as `2022-12-14` will be converted to `2022-12-14 00:00`). Similar time setting is implemented for the following formats: `LocalDateTime`, `Timestamp`, `Instant`, `OffsetDateTime`. 
```

Comment:
added support for using datetime formats without time (e.g. `yyyy-MM-dd`) for types that contain time (`LocalDateTime`, `Timestamp`, `Instant`, `OffsetDateTime`). Missing time values are filled with zeros, e.g. when sending `2022-12-14` as `LocalDateTime` parameter, the conversion will result in `2022-12-14 00:00`.